.github/CODEOWNERS

```diff
@@ -94,6 +99,11 @@
 /kernel/byterun/  @maximedenes
 # Secondary maintainer @silene
 
+/kernel/sorts.ml{,i} @SkySkimmer
+/kernel/uGraph.ml{,i} @SkySkimmer
+/kernel/univ.ml{,i} @SkySkimmer
```

Comment:
Ultimately, I'd like to regain "ownership" of those files, but with our current respective knowledge, it is indeed better if @SkySkimmer can take care of it.